workflow/scripts/realisation_to_srf.py

```diff
@@ -478,9 +499,20 @@ def generate_fault_srfs_multi(
     environment.srf_directory.mkdir(exist_ok=True)
     environment.gsf_directory.mkdir(exist_ok=True)
     params.velocity_model_1d.write_velocity_model(environment.velocity_model_path)
+
+    # In practice, a Point source will be the only item in `faults`, so if there
+    # is a Point source, restrict to `single_threaded` mode for simplicity.
+    for name in faults:
+        if isinstance(faults[name], Point):
+            single_threaded = True
+            break
+
```

Comment:
I don't like this check because it would be better if it just checked for a single fault instead 
```suggestion
    single_threaded = single_threaded or len(faults) == 1
```